CRM/Activity/BAO/Activity.php

```diff
@@ -990,29 +991,29 @@ public static function createEmailActivity($sourceContactID, $subject, $html, $t
    * @param $html
    * @param string $emailAddress
    *   Use this 'to' email address instead of the default Primary address.
-   * @param int $userID
+   * @param null $userID
```

Comment:
I'm not clear on this - did the IDE do it?

Reply:
Seems like it - fixed

Reply:
Thanks. Although $cc and $bcc are strings. Also $additionalDetails.

Reply:
If only there was some language feature that let you specify the type, like in C... (grin)

Reply:
aha - I think I just won the good fight against the IDE